### DISCUSSION TITLE
Put on different workspace if `onworkspace:1`

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [hyprwm/Hyprland#4232] [Q&A]
`onworkspace (how many windows are on the workspace) - int`
With this new feature is it possible to do anything like this ?
﻿
`windowrulev2 = workspace +1 ,onworkspace:1,workspace special:music_player`
﻿
To put the window in the next workspace if the current one has an open window in it

**Comment by vaxerski** [comment]
no, workspace is not dynamic

**Reply by LamprosPitsillos** [reply]
`workspace +1` ?

**Reply by vaxerski** [reply]
`workspace +1` is the `workspace` windowrule which is not dynamic aka only fires when the window is opened
